README.md

```diff
@@ -1,2 +1,2 @@
 CoddingChallengeSergioAndGabriel
```

Comment:
This README file is not as descriptive as it should be. Please gently use the template of the README.md file given in the course.